src/SortableTableHeader.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSort, faSortUp, faSortDown } from '@fortawesome/free-solid-svg-icons';
+
+const SortableTableHeader = ({ sortConfig, onSortColumn, colName }) => {
+    // no columns have been sorted yet, or the last sorted column isnt itself
+    if (sortConfig['columnKey'] !== colName) {
+        return (
+            <button type='button' onClick={() => onSortColumn(colName)}>
+                {colName}
+                <FontAwesomeIcon icon={faSort} />
+            </button>
+        )
+    } else {
+        return (
+            <button type='button' onClick={() => onSortColumn(colName)}>
+                {colName}
+                {sortConfig['dirToSort'] === 'asc' ?
+                    <FontAwesomeIcon icon={faSortDown} /> :
+                    <FontAwesomeIcon icon={faSortUp} />
+                }
+            </button>
+        )
+    }
```

Comment:
Seems like the only difference is the icon name.
I would move the logic outside of rendering logic.


```suggestion
    let icon = faSort;
    if (sortConfig['columnKey'] === colName) {
      icon = sortConfig['dirToSort'] === 'asc' ? faSortDown : faSortUp
    }
    return (
        <button type='button' onClick={() => onSortColumn(colName)}>
            {colName}
            <FontAwesomeIcon icon={icon} />
        </button>
    )
```